tests/Fixtures/Adapter/TestAdapter.php

```diff
@@ -249,16 +249,31 @@ public function closeIssue($id)
      */
     public function createComment($id, $message)
     {
-        return 'https://github.com/gushphp/gush/issues/'.$id.'#issuecomment-1';
+        $url = 'https://github.com/gushphp/gush/issues/'.$id.'#issuecomment-2';
+        $this->pullRequest['comments'][2] = [
+            'id' => 2,
+            'url' => $url,
+            'body' => $message,
+            'user' => 'phansys',
+            'created_at' => new \DateTime('1969-12-31T10:00:00+0100'),
+            'updated_at' => new \DateTime('1969-12-31T10:00:00+0100'),
+        ];
+
+        return $url;
     }
 
     /**
      * {@inheritdoc}
```

Comment:
i know 👍 
Gush will never be documented in this way because it is not a library neither a framework. For this reason i am strongly considering removing those.